### ISSUE TITLE
[Task] Migrate usage of AWS SDK v1 in integration tests

### ISSUE THREAD

**Issue by cPu1** [weaveworks/eksctl#5099]
As part of our effort to [migrate usage of aws-sdk-go (v1) to aws-sdk-go-v2](https://github.com/weaveworks/eksctl/issues/3215), the integration tests should also use aws-sdk-go-v2 for AWS API calls. This issue covers both configuring the SDK in integration tests, and replacing usage of SDK v1 AWS API calls with SDK v2.

**Comment by aclevername** [comment]
I've been making those changes as part of the regular tickets, do we need a separate ticket?

**Comment by aclevername** [comment]
https://github.com/weaveworks/eksctl/pull/5001/files#diff-36b539081bb77fa4b45ef7135b77d247f6b7792c640e5782f796da9e7383f200

**Comment by Skarlso** [comment]
Parts of this is actually blocked on the EKSAPI being migrated.

**Comment by Skarlso** [comment]
Blocking piece https://github.com/weaveworks/eksctl/issues/4884
